bigtable/admin.go

```diff
@@ -2150,9 +2150,60 @@ func (ac *AdminClient) RestoreTableFrom(ctx context.Context, sourceInstance, tab
 	return longrunning.InternalNewOperation(ac.lroClient, op).Wait(ctx, &resp)
 }
 
+type backupOptions struct {
```

Comment:
ExpireTime is a required field and not optional. Adding it to backupOptions indicates otherwise
https://github.com/googleapis/google-cloud-go/blob/649c075d5310e2fac64a0b65ec445e7caef42cb0/bigtable/admin/apiv2/adminpb/table.pb.go#L1367-L1374

Reply:
yeah, we can rename `backupOptions` if that's the issue

my suggestion was to have all configurable fields for a backup in a single struct

Reply:
It's not unreasonable to subsume expiration time into the set of variadic options if you explicitly validate the backupOptions has any necessary fields in CreateWithOptions.

The other way forward might would be if there's a reasonable default expiration (7d, etc).  However in that case I'd probably recommend that the service makes those behavioral choices via an optional field, and documents the defaults. Otherwise you end up with a potential consistency problem across client implementations.